### PR TITLE
FIX: cant reliably pay lnurl invoices when current currency is fiat (…

### LINKED ISSUE #5120: Bluewallet can't read QR-Code from BTCPAYSERVER correctly
* Many Phones tested
* BlueWallet app version  6.3.1
* self-test passes? yes

QR Code INVOICE from BTCPAYSERVER (Lighning) says 6€
Scanning with Bluewallet retrieves 5,68 €
After sending I get the attached error
With "Wallet of Satoshi" it works fine

Screenshot_20221015-102839
Screenshot_20221015-102905

### PR DESCRIPTION
**Pull request by Overtorment**
…closes #5120)

### CODE REVIEW

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: screen/lnd/lnurlPay.js [hunk 1]
```diff
@@ -73,13 +81,21 @@ const LnurlPay = () => {
 
   useEffect(() => {
     if (payload) {
-      let newAmount = payload.min;
+      /** @type {Lnurl} */
+      const LN = _LN;
+      let originalSatAmount;
+      let newAmount = (originalSatAmount = LN.getMin());
+      if (!newAmount) {
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply screen/lnd/lnurlPay.js:88]: Looks lime if its undefined or null or then parseInt will return 0

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/lry646nagrthk63qyyidlg5gci?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10632)

File: screen/lnd/lnurlPay.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Overtorment) [reply screen/lnd/lnurlPay.js:88]: i think not, lnurl-pay was specifically created as a workaround for zero-amount invoices

**Review by marcosrdz — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**